templates/activity/parts/summon-changes.hbs

```diff
@@ -1,19 +1,19 @@
 <section class="tab activity-{{ tab.id }} {{ tab.cssClass }}" data-tab="{{ tab.id }}" data-group="{{ tab.group }}">
     <fieldset>
         <legend>{{ localize "DND5E.SUMMON.CreatureChanges.Label" }}</legend>
-        <p>{{ localize "DND5E.SUMMON.CreatureChanges.Hint" }}</p>
-        {{ formField fields.match.fields.proficiency value=source.match.proficiency }}
+        <p class="hint">{{ localize "DND5E.SUMMON.CreatureChanges.Hint" }}</p>
```

Comment:
Looks like this causes a layout problem on Firefox because of an interaction between `flex-direction: column` and core's `flex: 0 0 100%;` on `.hint`:

<img width="513" alt="Screenshot 2024-08-22 at 14 06 49" src="https://github.com/user-attachments/assets/366fb62b-d6c7-4b64-a5d3-3ad5d6b30267">

Can be solved by wrapping the hint in `<div class="form-group"></div>`